README

```diff
@@ -1,27 +1,26 @@
 systemd System and Service Manager UI
 
 DETAILS:
-        http://0pointer.de/blog/projects/systemd.html
+        https://0pointer.de/blog/projects/systemd.html
 
 WEB SITE:
-        http://www.freedesktop.org/wiki/Software/systemd
+        https://systemd.io
 
 GIT:
-        git://anongit.freedesktop.org/systemd/systemd-ui
-        ssh://git.freedesktop.org/git/systemd/systemd-ui
+        https://github.com/systemd/systemd-ui.git
+        git@github.com:systemd/systemd-ui.git
```

Comment:
Please keep the `ssh://` here for explicitness.

Reply:
Sure, fixed. :-)